src/com/theironyard/novauc/ReadWriteJson.java

```diff
@@ -0,0 +1,67 @@
+package com.theironyard.novauc;
+
+import jodd.json.JsonParser;
+import jodd.json.JsonSerializer;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Created by Eric on 2/15/17.
+ */
+
+
+public class ReadWriteJson {
+
+
+
+    public static void readingRainbow() throws IOException {
+        File bookFile = new File("bookFile.json");
+        Scanner bookScanner = new Scanner(bookFile);
+        bookScanner.useDelimiter("\\Z");
+        String contents = bookScanner.next();
+
```

Comment:
Eric, I needed to add a the line while (bookScanner.hasNext()) { above line 23 otherwise I got an exception.  This probably was not the case for you since you probably had something in your bookFile.json file from the start.  However if there was nothing in that file this line is needed to avoid an exception, due to that fact that you are trying to read something that is not there on line 23.  There is no way you could have known this because I'm sure it works fine on your machine.  So there is no need to resubmit, however if you don't understand the reasoning behind why that line is needed please talk to me in class.  Otherwise, you did a great job on this assignment.  Your architecture was really nicely done and the use of try catch blocks is very well done.